cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help

```diff
@@ -43,6 +43,9 @@ OPTIONS
        --debug
            Run in remote debugging mode.
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars created while building.
```

Comment:
This wouldn't apply when `ballerina run` is used with a JAR right?

Should we say something like

```suggestion
           Package the Choreo observability extension in the executable when run is used with a source file or a module.
```

Reply:
Please check the line length too. IIRC, the maximum we used to have was 80.

Reply:
Yes these flags are applicable only when the commands are run against bal files.

Reply:
+1

Reply:
Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5